tests/testerina-integration-test/src/test/resources/testng.xml

```diff
@@ -46,6 +46,7 @@ under the License.
             <class name="org.ballerinalang.testerina.test.ImportTest" />
             <class name="org.ballerinalang.testerina.test.ModuleExecutionTest" />
             <class name="org.ballerinalang.testerina.test.CodeCoverageReportTest" />
+            <class name="org.ballerinalang.testerina.test.SourcelessTestExecutionTests"/>
```

Comment:
I think the 'SourcelessTestExecutionTests' file isn't committed.

Reply:
Added SourcelessTestExecutionTests